ebpf/c/tracer-ebpf.c

```diff
@@ -556,6 +537,8 @@ static int increment_udp_stats(struct sock *sk,
 		}
 
 		struct ipv4_tuple_t t = {};
+		t.metadata = CONN_TYPE_UDP;
+
 		if (!read_ipv4_tuple(&t, status, sk)) {
```

Comment:
It could be nice to add a connectionType parameter to `read_ipv{4,6}_tuple()` and set the field in the function. That way we don't need to remember to initialize each tuple with a connection type

Reply:
Good idea, will do that 👍 , however there is structs for which we don't use `read_ipv{4,6}_tuple()` so those will still need to initialize the type manually (when we try to map an ipV6 typle to an ipV4 tuple for instance)